test/swift_class_test.exs

```diff
@@ -232,47 +272,47 @@ defmodule SwiftClassTest do
     test "to_atom" do
       input = "buttonStyle(style: to_atom(style))"
 
-      output = [["buttonStyle", [["style", {:to_atom, [], [{:style, [], Elixir}]}]], nil]]
+      output = {:buttonStyle, [], [[style: {Elixir, [], {:to_atom, [], [{Elixir, [], :style}]}}]]}
 
       assert parse(input) == output
     end
 
     test "to_integer" do
       input = "frame(height: to_integer(height))"
 
-      output = [["frame", [["height", {:to_integer, [], [{:height, [], Elixir}]}]], nil]]
+      output = {:frame, [], [[height: {Elixir, [], {:to_integer, [], [{Elixir, [], :height}]}}]]}
 
       assert parse(input) == output
     end
 
     test "to_float" do
       input = "kerning(kerning: to_float(kerning))"
 
-      output = [["kerning", [["kerning", {:to_float, [], [{:kerning, [], Elixir}]}]], nil]]
+      output = {:kerning, [], [[kerning: {Elixir, [], {:to_float, [], [{Elixir, [], :kerning}]}}]]}
 
       assert parse(input) == output
     end
 
     test "to_boolean" do
       input = "hidden(to_boolean(is_hidden))"
 
-      output = [["hidden", [{:to_boolean, [], [{:is_hidden, [], Elixir}]}], nil]]
+      output = {:hidden, [], [{Elixir, [], {:to_boolean, [], [{Elixir, [], :is_hidden}]}}]}
 
       assert parse(input) == output
     end
 
     test "camelize" do
       input = "font(family: camelize(family))"
 
-      output = [["font", [["family", {:camelize, [], [{:family, [], Elixir}]}]], nil]]
+      output = {:font, [], [[family: {Elixir, [], {:camelize, [], {Elixir, [], :family}}}]]}
 
       assert parse(input) == output
     end
 
     test "snake_case" do
       input = "font(family: snake_case(family))"
 
-      output = [["font", [["family", {:snake_case, [], [{:family, [], Elixir}]}]], nil]]
+      output = {:font, [], [{Elixir, [], {:snake_case, [], [{Elixir, [], :family}]}}]}
```

Comment:
Line 307: Is the `camelize` function missing brackets on the arguments?
On line 315 and elsewhere, we expect a list of arguments.